### ISSUE TITLE
Stroke thickness

### ISSUE THREAD

**Issue by lamelama07** [Pullusb/GP_onion_peel#1]
Dear Samuel Bernou,

First of all, thank you for the amazing work with all your add-ons. I was wondering if changing the onion peels to use the screen space instead of the world space option as the default stroke thickness is possible. Currently, if I draw with the previous setting, the thickness swaps back to the world space option whenever the onion peel does a refresh.  

Best regards,
Mark Laszlo

**Comment by Pullusb** [comment]
Sorry I never took time to answer your question.
I got involved on lots of other project and forgot about it... :sweat: 
I saw you closed the issue. Have you found a fix or a workaround to your problem ?

If it's still a thing I'll need to test to really grasp the essence of it.
Feel free to reopen issue if needed

Happy you appreciate my addons.
Regards,
S.

**Comment by lamelama07** [comment]
Hi,

the simplest workaround was to set the stroke thickness back to the default world space, and then to choose a thickness multiplier value, which was similar to the screen space thickness I used to animate in. 

After that, I switched back the stroke thickness to screen space, to continue my work. That way when the peels refreshed, they used  the "correct" thickness world space multiplier value.
I do not know why I could not come up with this earlier. It is not a proper solution, but works perfectly.  

Regards,
Mark
